### DISCUSSION TITLE
Edit vercel template description

### DISCUSSION THREAD

**Discussion by nisabmohd** [vercel/community#7263] [Help] [answered]
### Summary

I have recently published my documentation template on Vercel Templates, but I noticed there is a mistake in the description. Could someone help me correct it? Here is the link to the template: https://vercel.com/templates/next.js/documentation-template.

### Steps to Reproduce

 I would like my template description to be revised and maintained as follows:
> This feature-packed documentation template, built with Next.js, offers a sleek and responsive design, perfect for all your project documentation needs.

**Comment by leerob** [comment, marked as answer]
Thank you, updated: https://vercel.com/templates/next.js/documentation-template

[a comment by an automated account is not shown]
